style.css

```diff
@@ -505,6 +587,22 @@ footer > a {
   align-items: center;
 }
 
+@media (min-width: 1024px) {
+  /* prettier-ignore */
+  .about-myself {
+    background: url(./assets/about_me_desktop_left.svg) no-repeat 0% 20%, url(./assets/about_me_desktop_rigth.svg) no-repeat 100% 0%;
+  }
+}
```

Comment:
- [ ] Kindly you can add `background-size: 13vw;` here to make the image not come over the text.

![image](https://user-images.githubusercontent.com/35535130/159818843-3da63d94-7595-42d6-9ac7-5be26152560a.png)